src/main/kotlin/com/tradeshift/blayze/Model.kt

```diff
@@ -95,25 +97,25 @@ class Model(
      *
      * @return predicted outcomes and their probability, e.g. {"positive": 0.3124, "negative": 0.6876}
      */
-    fun predict(inputs: Inputs): Map<Outcome, Double> {
+    fun predict(inputs: Inputs, pseudoCounts: Map<FeatureName, PseudoCount> = mapOf()): Map<Outcome, Double> {
```

Comment:
A Model has many Features. Only some of those have any notion of PseudoCounts, so make it general

Reply:
You need to make some kind of PredictParams class that you can pass here.
```kotlin
data class PredictParams (
    val text = Map<String, Multinomial.PredictParams>,
    val categorical = Map<String, Multinomial.PredictParams>,
    val gaussian = Map<String, Gaussian.PredictParams>
)
```
Similarly for UpdateParams